charts/kong/README.md

```diff
@@ -229,11 +229,16 @@ Kong can be configured via two methods:
 
 ### Separate admin and proxy nodes
 
+*Note: although this section is titled "Separate admin and proxy nodes", this
+split release technique is generally applicable to any deployment with
+different types of Kong nodes. Separating Admin API and proxy nodes is one of
+the more common use cases for splitting across multiple releases, but you can
+also split releases for hybrid mode CP/DP nodes, split proxy and Developer
+Portal nodes, etc.*
+
 Users may wish to split their Kong deployment into multiple instances that only
-run some of Kong's services, e.g. where some nodes only run the proxy and other
-only run the admin API, or where some nodes only run Developer Portal services.
-These require separate Helm releases (i.e. you run `helm install` once for
-every instance type you wish to create).
+run some of Kong's services (i.e. you run `helm install` once for every
+instance type you wish to create).
```

Comment:
This is intentional since there's not really a great way to name this feature such that it's obvious why you'd use it. Separating admin and proxy is the most common based on anecdotal experience from community questions, so I used that as the title to grab those users' attention.

Historically the others (separating the Enterprise services) was something we'd also expect questions to come through customer support and sales engineering, who can direct those users to the right section, whereas the community users need to find it on their own.

Reply:
Like many of the comments elsewhere, I'd need to see evidence from elsewhere that it's a problem, or agree that the proposed change still meets the intent of the original also, and I don't here.

My expectation is that users will often read it starting from a specific question (here, "how do I separate my Kong admin and proxy into separate instances?"), and the section title needs to make it discoverable for them, rather than trying to achieve some ideal of title correctness. There's no official name for these as a group--we don't refer to "Kong functions" or "Kong services" much elsewhere, and instead refer to them individually by their own names, e.g. in the main doc site where they [are adjacent, but not under a unified header](https://i.imgur.com/V2z5Sr7.png). We know they're similar because we're familiar with the implementation, but can't expect that users necessarily do.

To try and spin an analogy, were this document a pharmacy, we (the pharmacy owners) likely would not have shelf sections labeled "μ-opioid receptor agonists" and "non-steroidal anti-inflammatory COX enzyme inhibitors" even though those are entirely accurate and technically specific descriptions--customers are much more likely to find what they're looking for if those sections are "gastric distress relief" and "pain relief", even though the "pain relief" section neither contains all known pain relievers, nor do its contents solely relieve pain. We also have staff (SEs and CSEs, and community forums) available to direct customers if they need something in those sections for a purpose other than its most common use--much like we can direct users looking to separate out Portal services or manage their CP/DP nodes, the pharmacy staff can direct customers to the NSAID aisle if they need something to deal with a fever.

Reply:
Alright. I think we're hitting the "reference" vs "how-to" difference here, which I've explained in https://github.com/Kong/charts/pull/160#discussion_r445179942.